test/utils/valideringUtilsTest.ts

```diff
@@ -71,4 +75,38 @@ describe("valideringUtils", () => {
       });
     });
   });
+
+  describe("validerVideoLink", () => {
+    it("validate incorrect url format, returns error message", () => {
+      const validationMessage = validerVideoLink("https://invalid.url");
+
+      expect(validationMessage).to.equal(texts.invalidVideoLink);
+    });
+    it("validate correct url format and no whitespace, doesn't return error message", () => {
+      const validationMessage = validerVideoLink("https://video.nav.no/abc");
+
+      expect(validationMessage).to.be.undefined;
+    });
+    it("validate url with whitespace, returns error message", () => {
+      const validationMessage = validerVideoLink(
+        "https://video.nav.no/abc space"
+      );
+
+      expect(validationMessage).to.equal(texts.whiteSpaceInVideoLink);
+    });
+    it("validate url with whitespace only before and after valid url, doesn't return error message", () => {
+      const validationMessage = validerVideoLink(
+        "   https://video.nav.no/abc  "
+      );
+
+      expect(validationMessage).to.equal(undefined);
```

Comment:
Er det forskjell på `.to.be.undefined` og `.to.equal(undefined)` ? 🧐

Reply:
Hmm, det vet jeg ikke 🤔 Det er en glipp, jeg tenkte `to.be.undefined` begge steder! Jeg tenker den er grei å bruke når den først finnes 👍